udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -1,11 +1,12 @@
 {% extends theme('layouts/1-column.html') %}
```

Comment:
Seems like we have an infinite vertical scroll on the dataset page

Reply:
fixed to prevent this for js enabled tabs